packages/halo-theme/src/custom-elements/ef-calendar.less

```
@@ -155,6 +155,10 @@
   [part~='selection'] {
     outline: none;
     border: 1px solid @calendar-cell-background-color;
+
+    ::slotted(*) {
```

If result is the same, it might easier to reading with `&`.
```suggestion
    &::slotted(*) {
```

---

The result after compile is not the same.